src/main/java/com/ontotext/kafka/error/LogErrorHandler.java

```diff
@@ -15,6 +18,8 @@
 import java.util.Properties;
 
 public class LogErrorHandler implements ErrorHandler {
```

Comment:
I know it is not part of this change, but please add why we use a custom error handler as javadoc and not using the default one.